README.md

```diff
@@ -1,5 +1,36 @@
-# word-lapse
+# Word Lapse
 
-WORK IN PROGRESS - Explore how a word changes in meaning over time
+Explore how a word changes over time
 
 <img src="https://raw.githubusercontent.com/greenelab/word-lapse/main/app/public/share-thumbnail.jpg?raw=true" width="300px">
+
+[**✨ OPEN THE APP ✨**](https://greenelab.github.io/word-lapse/)
+
+### How does it work?
+
+Type in a word, and Word Lapse will tell you things about it that have changed per year, like associated words or frequency of use.
+
+The information is computed from a XXX machine learning model by XXX.
+The model was trained on over XXX abstracts and full-texts from XXX.
```

Comment:
```suggestion
Word2Vec is a natural language processing (NLP) model that represents words in dense vectors (300 dimensions).
This model constructs these dense vectors by training a shallow neural network to accomplish the NLP task of predicting a word given their neighboring words.
Once trained, every unique word is now associated with a dense vector, which can be used for downstream analyses.

Word2Vec model is unstable, and we took an intra-year and inter-year approach to account for this instability.
We trained ten different Word2Vec models on documents sorted by each publication year.
Next, we generated combination pairs for every model trained within a given year and calculated the intra-year distance, which consists of calculating the [cosine distance](https://medium.datadriveninvestor.com/cosine-similarity-cosine-distance-6571387f9bf8) for every token within each model pair
Following intra-year distance, we calculated the inter-year distance by taking the same approach described above for two different years rather than the same year.
Lastly, we took the ratio of inter-year distance over intra-year distance and any values over a threshold is considered to be a changepoint.
```

Reply:
In the LICENSE file, may I recommend adding `BSD 3-Clause License` as the top line of the file. Unless there's a change in the file that makes it _not_ BSD-3, it's not clear which license it is at a glance.

Reply:
I just copied and pasted this from the default GitHub templates. We can try adding it, I just want to make sure that GitHub can accurately detect it, because they show a nice icon and license summary in the GitHub UI.